src/docs/components/tools/vendorCookieInspector/vendorCookieInspector.jsx

```diff
@@ -123,13 +123,37 @@ export default class VendorCookieDecoder extends Component {
 		this.setState({ localCookie: readCookie(VENDOR_CONSENT_COOKIE_NAME) });
 	};
 
+	readParams = () => {
+		var vendor_cookie = this.getUrlParam("string");
```

Comment:
The name `string` is a little ambiguous. Maybe make it `consent-string` or `consent-data` or `base64-value` or something specific to this page.